### DISCUSSION TITLE
Please let me know if this is a good or bad idea

### DISCUSSION THREAD

**Discussion by noni12131** [uBlockOrigin/uBlock-issues#3011] [Ideas]
I see that youtube is cracking down a lot, what if you made it where it changes the playback video speed, and skips through the ads to the video. They started blocking me on firefox, maybe it would be harder for them to detect if you try that method, I know hulu has been using it for quite awhile. I hope you do see this, and thank you so much have a great day night evening!

I had Started a video youtube blocks me from playing it, even after reinstalling U block at first.

I feel like youtube wants to keep this back and forth idk I would keep that idea as a last resort, it did happen a bit for me today it seems it was fixed by 1:31 am cst, it seems you guys are keeping ahead with it so far tho

**Comment by noni12131** [comment]
Ad got through 1:50am
![rock](https://github.com/uBlockOrigin/uBlock-issues/assets/153148082/56931578-ac3c-418c-a25a-1d108dd0e474)
![asdada](https://github.com/uBlockOrigin/uBlock-issues/assets/153148082/0fee207d-3fb4-493a-9e53-87221a63652d)

**Comment by ghost** [comment]
https://github.com/uBlockOrigin/uAssets/issues/20586
